### ISSUE TITLE
Top Nav mouse-over menus are not accessible

### ISSUE THREAD

**Issue by dajoturner** [ucsc/webtemplates2014#68]
Len Burns reported this issue. "The page contains a menu activated by a mouseover, the "Undergraduate," link.  This is completely inaccessible to a screen reader user.  In fact, without close scrutiny, its existence would go unnoticed. This would also present severe problems for other users of adaptive technology who rely upon the keyboard.  One good test strategy for developers is to unplug the mouse and explore a page using only the keyboard.  The question is, can the page still be fully utilized, or are parts inaccessible or contain areas where a keyboard user will become trapped."

**Comment by thomasgardner** [comment]
In the UCSC Technology Accessibility and Civil Rights in Higher Education workshop for Global Accessibility Awareness Day (5-21-2015) workshop leader Don Barrett, Assistive Technology Specialist, US Department of Education / Office for Civil Rights mentioned that this is still an issue in both the 1:00 - 3:00 pm and the 3:30 - 5:30 pm sessions.

**Comment by knice** [comment]
@thomasgardner Are you sure you don't mean #284? Len's issue here was that the links were **completely inaccessible** from the keyboard. I [fixed that issue](https://github.com/ucsc/webtemplates2014/issues/68#issuecomment-53501815). Issue #284 covers the other problem, which is that the dropdowns need to be visible when tabbed to.

**Comment by thomasgardner** [comment]
@knice Yes it is #284. This proves I'm not good at multitasking. I was trying to pay attention to the speaker, search GitHub, etc. and I did not respond to the correct issue. Thanks for checking. I'll make notes in #284.

**Comment by luckyluke007** [comment]
Hello!

 

New message, please read <http://notisector.com/twice.php?87y1y>

 

trung@scgstudio.com

**Comment by luckyluke007** [comment]
Hello!

 

New message, please read <http://elite-massage.com.au/until.php?m>

 

trung@scgstudio.com
